### PR TITLE
Feat/event invites notifications

### PR DESCRIPTION
**Pull request by u22520016**
This code allows users to see a list of events other users invite the current user to.  The three buttons do the following:
1. Accept - Accepts the invitation and moves the user's ID from the invited array to the joined array in the database.
2. Decline - Declines the event invitation and removes the user's uid from the invited array in the database
3. Stay notified - The user subscribes to the event rather than joining and moves the user's uid from the invited array to the subscribed array.

Steps to test:
1. Comment out the current body on the feed page
2. Add the EventInvitesList component to the body of the feed page
3. Run the app with a user who is invited to a few events.

(I know the UI looks super ugly for now.  But I will change that in a separate branch)

### CODE REVIEW

File: lib/services/events_S/event_service.dart [hunk 1]
```diff
@@ -109,6 +129,21 @@ class Events {
...
   }
 
+  Future<void> declineEventInvitation(Event event) async {
+    final DocumentReference docRef = FirebaseFirestore.instance.collection('events').doc(event.eventID);
+
+    try {
+      final currentUser = FirebaseAuth.instance.currentUser;
+      if (currentUser != null) {
+        await docRef.update({
+          'invited_users': FieldValue.arrayRemove([currentUser.uid])
```
Comment (MonikaTheiss) [inline lib/services/events_S/event_service.dart:139]: the field I used was just called "invited" not invited_users

Comment (u22520016) [reply lib/services/events_S/event_service.dart:139]: That's odd...it worked on my end...did it work on yours?

Comment (MonikaTheiss) [reply lib/services/events_S/event_service.dart:139]: no it says no new event invites on my side

Comment (u22520016) [reply lib/services/events_S/event_service.dart:139]: Ohhh...somehow only the first document has "invited_users".  Can I remove that and change the code to "invited"?

Comment (MonikaTheiss) [reply lib/services/events_S/event_service.dart:139]: yes please do

Comment (u22520016) [reply lib/services/events_S/event_service.dart]: I changed it to 'invite'

Comment (pieryvdl) [reply lib/services/events_S/event_service.dart]: I noticed on this document: AlV61ZYE6FARUAHT3xv1 taht there is both, could explain why it worked on Franco's side and not on the others. Nevertheless I see it has been updated.

**Review by pieryvdl — APPROVED** [review]
I get the exact same issue as Monika on one of my tournaments. However the rest of the code looks good. In the future we should look into adding feedback or making the component disappear when clicking 'decline' because it looks stuck... though if you move between pages it it fine. Just look at Monika's issue.

**Review by MonikaTheiss — APPROVED** [review]
Works perfectly now, thank you!

**Review by pieryvdl — APPROVED** [review]
Found it working on my side as well. Thanks Franco! You did good work. :)

**Pull request merged**